addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -160,3 +161,43 @@ def test_efaktur_total_rounding_accuracy(self):
 
         self.assertEqual(amount_untaxed_total, amount_untaxed_sum)
         self.assertEqual(amount_tax_total, amount_tax_sum)
+
+    def test_efaktur_do_not_consume_code(self):
```

Comment:
Nothing to be done, but I'm surprised this test class doesn't inherit from `AccountTestInvoicingCommon`, there'd be less boilerplate code

Reply:
It wasn't done by someone from the R&D accounting so it's most likely why (although not sure why nobody asked to change during the review).
I can change it if you prefer, but since this test file is not that big I think it's ok to not refactor atm